src/G2/Equiv/Verifier.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE MultiWayIf #-}
```

Comment:
Do we ever actually use this in the current version of the code?

Reply:
Yes, line 459
https://github.com/BillHallahan/G2/pull/93/files/9386ddffd23b2029dbc9986236f21df618d13e72#diff-c0e54ace2c495328574320cf6e6e6b5241fc2d732124711d046e31d57f0c1ccdR459-R462